packages/backend/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "backend",
   "scripts": {
-    "test": "jest --passWithNoTests --maxWorkers=50%",
+    "test": "jest --passWithNoTests --maxWorkers=50% --logHeapUsage",
```

Comment:
:eyes: 
Is this looking into the inconsistent `backend` test failures?
Does `--logHeapUsage` this need `--runInBand`?
https://jestjs.io/docs/cli#--logheapusage

Reply:
oh
- https://github.com/interledger/rafiki/pull/1132

Reply:
Yep, we had an out-of-memory exception earlier, so wanted to debug the tests